testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -1604,10 +1604,7 @@ QUnit.test("right key in submenu can move focus to next item of main menu (horiz
 
     visibleSubmenu = Submenu.getInstance(instance._visibleSubmenu.element());
 
-    $(visibleSubmenu._itemContainer())
-        .trigger(rightKeyKeydown)
-        .trigger(rightKeyKeydown)
-        .trigger(rightKeyKeydown);
+    $(visibleSubmenu._itemContainer()).trigger(rightKeyKeydown);
```

Comment:
@mrjohn0011 , please, take your attention to this part of code

Reply:
This change is ok. This menu test was wrong and 2 additional triggers was unnecessary